src/Aspire.Hosting/DistributedApplication.cs

```diff
@@ -194,12 +194,12 @@ public static IDistributedApplicationBuilder CreateBuilder(DistributedApplicatio
 
     private static void WaitForDebugger()
     {
-        if (Environment.GetEnvironmentVariable("ASPIRE_WAIT_FOR_DEBUGGER") == "true")
+        if (Environment.GetEnvironmentVariable(KnownConfigNames.WaitForDebugger) == "true")
```

Comment:
Is there a reason Environment is used directly in a couple of places instead of IConfiguration? cc @mitchdenny